### ISSUE TITLE
SLOI, SROI also need *_RV32 version; REV, REV8, ORC_B instruction definition.

### ISSUE THREAD

**Issue by davidmlw** [chipsalliance/rocket-chip#2549]
1. Like RORI has RORI_RV32 version, I think SLOI and SROI also need *_RV32 version.
https://github.com/chipsalliance/rocket-chip/blob/f1521796c292ee87958df6ea52f12145d869c83f/src/main/scala/rocket/Instructions.scala#L823

2. REV, REV8, ORC_B are instructions from Zbb, while bitmanip defines them as pseudo instructions. Is it hte case? where is the REV, REV8 and ORC_B spec definitions?

**Comment by aswaterman** [comment]
Right.  I'll update this stuff once the bitmanip task group makes decisions on what will be included.

Even though REV[8] and ORC.B are pseudoinstructions, it still helps to have them in this list, because some cores will implement only these forms, rather than implementing all of GORCI and GREVI.
